kedro-telemetry/pyproject.toml

```diff
@@ -37,11 +37,5 @@ readme = {file = "README.md", content-type = "text/markdown"}
 version = {attr = "kedro_telemetry.__version__"}
 
 [tool.isort]
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-use_parentheses = true
-line_length = 88
-known_third_party = "kedro"
-
-[tool.black]
+profile = "black"
+known_first_party = "kedro_telemetry"
```

Comment:
Because of how we're now installing the plugin itself instead of just it's requirements. Other plugins (like `kedro-airflow`) avoid this by using relative imports--probably by luck--but generally wouldn't prefer relative imports.

Probably needs to be a lot more consistency between team-maintained plugins on this.

Also, use of `profile` should be done in all the plugins.